### DISCUSSION TITLE
How to exclude subfolder

### DISCUSSION THREAD

**Discussion by TmPostPrtt** [offen/docker-volume-backup#391] [Q&A] [answered]
Hey, here an example of my problem : 
- MyFolder
  - Sub1
  - Sub2
  - Sub3, etc...
  - File1
  - File2
  - File3, etc...

I would like to exclude Sub54, is it possible ?

**Comment by m90** [comment, marked as answer]
You can use `BACKUP_EXCLUDE_REGEXP` which matches the full filepath:

```
# When given, all files in BACKUP_SOURCES whose full path matches the given
# regular expression will be excluded from the archive. Regular Expressions
# can be used as from the Go standard library https://pkg.go.dev/regexp

# BACKUP_EXCLUDE_REGEXP="\.log$"
```

Reference https://offen.github.io/docker-volume-backup/reference/#configuration-reference
